.idea/compiler.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
.idea folder shouldn't exist in project repo, use git-ignore plugin to ignore this files